mappings/net/minecraft/block/entity/vault/VaultServerData.mapping

```diff
@@ -0,0 +1,41 @@
+CLASS net/minecraft/unmapped/C_nsbycoiv net/minecraft/block/entity/vault/VaultServerData
+	FIELD f_bzryaxno itemsToEject Ljava/util/List;
+	FIELD f_gdjgafzz lastFailedInsert J
```

Comment:
```suggestion
	FIELD f_gdjgafzz lastFailedInsertTime J
```
or
```suggestion
	FIELD f_gdjgafzz lastFailedInsertAt J
```
if we want to follow mojang's convention from `"state_updating_resumes_at"`
(setter+getter need updating, too)

Reply:
I like time better than at. I wonder if we should change the other one too time as well

Reply:
They should probably match, yeah